mappings/net/minecraft/client/gui/screen/DeathScreen.mapping

```diff
@@ -1,3 +1,6 @@
 CLASS net/minecraft/class_364 net/minecraft/client/gui/screen/DeathScreen
 	FIELD field_1111 ticksSinceDeath I
+	FIELD field_13322 deathMessage Lnet/minecraft/class_1982;
```

Comment:
```suggestion
	FIELD field_13322 message Lnet/minecraft/class_1982;
```
deathMessage is redundant since this is the only message and its in the DeathScreen class